VersionControlUI/src/DiffManager.cpp

```diff
@@ -404,6 +404,49 @@ bool DiffManager::findChangedNode(Model::TreeManager* treeManager, Model::NodeId
 	return false;
 }
 
+void DiffManager::setOverlayInformationAccordingToChangeType(FilePersistence::ChangeType changeType,
+																	QString& highlightOverlayStyle, QString& highlightOverlayName,
+																	QString& arrowIconOverlayStyle, QString& arrowIconOverlayName,
+																	bool iconsForMoveAndModify)
+{
+	switch (changeType)
+	{
+		case FilePersistence::ChangeType::Deletion:
```

Comment:
Now that this is a separate method, I would explicitly "nullify" strings which should be null: `str = {}`